bitnami/mariadb-galera/values-production.yaml

```
@@ -555,6 +555,11 @@ metrics:
     ##
     # pullSecrets:
     #   - myRegistryKeySecretName
+  ## MySQL exporter additional command line flags
+  ## Can be used to specify command line flags, for example:
+  ##
+  ## extraFlags: "--collect.binlog_size"
+  ##
```

Could we use an array for this? IMHO it's easier to manage flags as an array:
```suggestion
  ## MySQL exporter additional command line flags
  ## Can be used to specify command line flags
  ## E.g.:
  ## extraFlags:
  ##   --collect.binlog_size
  ## 
  extraFlags: []
```

---

Yup, good idea, the changes look good to me! I'll add a small change to the README too.